public/img/test.html

```diff
@@ -1,15 +1,20 @@
-<!-- <html>
+<html>
   <body>
     <script>
       window.addEventListener("message", event => {
         console.log(event);
       });
+
+      const frameVar = document.getElementById('testFrame');
+      frameVar.contentWindow.postMessage(messageVariables2, "*");
+
     </script>
 
     <iframe
-      src="http://localhost:3000/dashboard-solo/new?orgId=1&from=1638940961704&to=1638962561704&panelId=2&downloadId=hello&messageTargetDomain=localhost:3000"
+      src="http://localhost:3000/d-solo/mKMsyIcnk/tash?orgId=1&var-testVar=5&from=1636538511887&to=1636539608910&panelId=2"
       width="1000"
       height="600"
+      id="testFrame"
     ></iframe>
   </body>
-</html> -->
+</html>
```

Comment:
Is this file supposed to be commited?

Reply:
This is just a useful way of being bale to test our iframes when deving. Easiest to just keep it in i reckon atm (At least until we improve/nail our dev setup for grafana work)